cypher-shell/src/integration-test/java/org/neo4j/shell/commands/CypherShellVerboseIntegrationTest.java

```diff
@@ -228,4 +229,22 @@ public void cypherWithOrder() throws CommandException {
             assertThat( actual, containsString( "n.age ASC" ) );
         }
     }
+
+    @Test
+    public void cypherWithExplainAndRulePlanner() throws CommandException {
+        //when
+        shell.execute("CYPHER planner=rule EXPLAIN MATCH (e:E) WHERE e.bucket='Live' and e.id = 23253473 RETURN count(e)");
```

Comment:
I noticed we don't have a `PR Tester with Neo4j 4.0`. I think this test would fail if we had one - rule planner is not supported in 4.0 any longer.

Reply:
yes lucky thing we don't test on 4.0. Lets fix this when we have done the docker migration and actually run test against 4.0

Reply:
But maybe we should not add any _new_ tests then, where we know they're gonna fail. Is there a different way we can expose the same bug with a test that will work in 4.0?

Reply:
No, `ColumnsLeft` is an old rule planner only thing.